lib/modules/platform/pr-body.ts

```diff
@@ -53,10 +53,63 @@ export function getRenovateConfigHashPayload(body: string): string | undefined {
   return match?.groups?.payload;
 }
 
+function getRenovateBodyIndexes(input: string | undefined | null): {
+  start: number;
+  startBody: number;
+  end: number;
+  endBody: number;
+} {
+  const inputBody = input ?? '';
+  // we want to remove all the new lines and spaces after the start tag and also before the end tag (\s*).
+  // use dedicated regular expressions for that
+  const startRegex = regEx(/<!--renovate:start-->\s*/);
+  const endRegex = regEx(/\s*<!--renovate:end-->/);
+  const renovateBodyIndexs: ReturnType<typeof getRenovateBodyIndexes> = {
+    start: 0,
+    startBody: 0,
+    end: inputBody.length,
+    endBody: inputBody.length,
+  };
+  const [startMatch, endMatch] = [
+    startRegex.exec(inputBody),
+    endRegex.exec(inputBody),
+  ];
+  if (startMatch) {
+    renovateBodyIndexs.start = startMatch.index;
+    renovateBodyIndexs.startBody = startMatch.index + startMatch[0].length;
+  }
+  if (endMatch) {
+    renovateBodyIndexs.endBody = endMatch.index;
+    renovateBodyIndexs.end = endMatch.index + endMatch[0].length;
+  }
+  return renovateBodyIndexs;
+}
+
+function getRenovateBody(input: string | undefined | null): string {
```

Comment:
```suggestion
function getRenovateBody(input: string): string {
```
validate on caller function